mlflow/store/artifact_repo.py

```diff
@@ -51,13 +60,17 @@ def list_artifacts(self, path):
         """
         pass
 
-    def download_artifacts(self, artifact_path):
+    def download_artifacts(self, artifact_path, dst_path=None):
         """
         Download an artifact file or directory to a local directory if applicable, and return a
         local path for it.
         The caller is responsible for managing the lifecycle of the downloaded artifacts.
-        :param path: Relative source path to the desired artifact
-        :return: Full path desired artifact.
+        :param path: Relative source path to the desired artifacts.
+        :param dst_path: Absolute path of the local filesystem destination directory to which to
+                         download the specified artifacts. This directory must already exist. If
+                         unspecified, the artifacts will be downloaded to a new, uniquely-named
+                         directory on the local filesystem.
+        :return: Absolute path of the local filesystem location containing the downloaded artifacts.
         """
```

Comment:
L79 should technically use the `get_path_module`, no?

Reply:
Good catch! Changed to `get_path_module()`